themes/default/content/learn/embedding-pulumi/getting-started/index.md

```diff
@@ -0,0 +1,234 @@
+
+With all of that in mind, let's imagine that the Pulumipus Boba Tea Shop, our company from the Fundamentals learning path, has been doing so well that we can expand to jump into other markets like selling tea blends or selling its boba tea automation application to other tea shops. We're now part of a cloud engineering platform team, and we need to make it easy for developers building our applications to work with the infrastructure we manage by making it something they don't need to think much about.
+
+This is exactly where Pulumi's Automation API can come in handy. The Automation API lets you embed Pulumi into applications and interfaces to empower others to get the infrastructure they need in the way you expect them to use it. You can also use it to work with CI/CD systems, chatops systems, monitoring systems, gitops systems, or anything else where you want to integrate the automation of the state of your infrastructure. For example, if you want your monitoring system to spin up additional spot instances or tear down unused spot instances based on traffic patterns without needing human interaction, you can use the Automation API to let alerts trigger infrastructure changes (perhaps with some policy as code to provide some guardrails to prevent the automation from tearing down everything or running up your bill).
+
+In our example scenario, let's pretend that we're starting to find development teams scaling across datacenters, regions, or zones. We're having trouble knowing what infrastructure is getting used, and we'd like to automatically stand up a small set of serverless functions for every region we enter so new infrastructure reports back to a central dashboard. Since we want this infrastructure to deploy without any human action, we can't use a standard CLI-based workflow like we would everywhere else. We're going to use the Automation API to integrate the creation of that serverless function infrastructure into every pipeline that stands up something new. For our function's output, we're going to gather bits of data related to time like timezones, time drift, and more. We'll set up a small monitoring function that reports this data on deployment without needing a lot of time, memory, or space to run (and therefore is cheap to use).
+
+Let's get started!
+
+## Setting up
+
+We're going to automate the use of Pulumi's CLI so we can use it in pipelines, APIs, and web interfaces. For the Pulumi program we're using to test our automation, we'll get to some sample code in a moment. If you'd like to use your own Pulumi program, however, go for it!
```

Comment:
What would you think of tweaking this phrasing a bit to play up the fact that we're running Pulumi programmatically ? (I ask because right now, it sounds a bit too much like we'll just be "shelling out" to Pulumi.)

```suggestion
We're going to embed Pulumi in a Python program so we can use it in pipelines, APIs, and web interfaces. For the Pulumi program we're using to test our automation, we'll get to some sample code in a moment. If you'd like to use your own Pulumi program, however, go for it!
```